### DISCUSSION TITLE
For my school paper: KubeEdge vs K8s

### DISCUSSION THREAD

**Discussion by LukasBloemer** [kubeedge/kubeedge#5362] [Q&A]
Hello,
I am writing a paper for school on the technical differences of KubeEdge compared to K8s and what advantages or disadvantages come with that. Unfortunately, I haven't found much on that topic so far. Can anyone give me some insights on what I should be looking for?
Any help is really appreciated.
Thanks in advance.

**Comment by ravaga** [comment]
Hello, you can take a look to this paper we wrote about the moving cloud-native workloads to the edge: https://www.mdpi.com/1424-8220/23/4/2215

You'll find some useful content in **section 6. Container Orchestration Tools for the Edge**, where we compare different alternatives to deploy K8s in the edge.
